docs/administration-guide/06-collections.md

```diff
@@ -1,35 +1,61 @@
 ## Creating Collections for Your Saved Questions
 ---
 
-Collections are a great way to organize your saved questions and decide who gets to see and edit things. Collections could be things like, "Important Metrics," "Marketing KPIs," or "Questions about users." Multiple [user groups](05-setting-permissions.md) can be given access to the same collections, so we don't necessarily recommend naming collections after user groups.
+![Collection detail](images/collections/collection-detail.png)
+
+Collections are a great way to organize your saved questions and decide who gets to see and edit things. Collections could be things like, "Important Metrics," "Product Team," "Marketing KPIs," or "Questions about users." Collections can even contain other collections, allowing you to create an organizational structure that fits your team. You can also choose which user groups should have what level of access to your collections (more on that below).
+
+Metabase starts out with a default top-level collection which is called "Our analytics," which every other collection is saved inside of.
 
 This page will teach you how to create and manage your collections. For more information on organizing saved questions and using collections, [check out this section of the User's Guide](../users-guide/06-sharing-answers.md).
 
 ### Creating and editing collections
-Only administrators of Metabase can create and edit collections. From the Questions section of Metabase, click on the `Create a collection` button. Give your collection a name, choose a color for it, and give it a description if you'd like.
+If a user has Curate access for a collection, they can create new sub-collections inside it and edit the contents of the collection. From the detail view of any collection, click on the `Create a collection` button to make a new one. Give your collection a name, choose where it should live, and give it a description if you'd like.
+
+![Create collection](images/collections/create-collection.png)
+
+By default, new collections will have the same permissions settings as the collection it was created in (its "parent" collection), but you can change those settings from the Edit menu.
+
+### Pinning things in collections
+![Pins](images/collections/pinned-items.png)
+
+One great new feature in Metabase is that you can now pin the most important couple of items in each of your collections to the top. Pinning a dashboard or question turns it into a big, eye-catching card that will help make sure that folks who are browsing your Metabase instance will always know what's most important.
```

Comment:
Dunno if we should describe things as a "new" feature in the docs just to make these a bit more evergreen.